seacrowd/sea_datasets/vimmrc/vimmrc.py

```diff
@@ -0,0 +1,198 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import (SCHEMA_TO_FEATURES, TASK_TO_SCHEMA,
+                                      Licenses, Tasks)
+
+_CITATION = """
+@ARTICLE{vimmrc,
+    author={Nguyen, Kiet Van and Tran, Khiem Vinh and Luu, Son T. and Nguyen, Anh Gia-Tuan and Nguyen, Ngan Luu-Thuy},
+    journal={IEEE Access},
+    title={Enhancing Lexical-Based Approach With External Knowledge for Vietnamese Multiple-Choice Machine Reading Comprehension},
+    year={2020},
+    volume={8},
+    pages={201404-201417},
+    doi={10.1109/ACCESS.2020.3035701}}
+"""
+
+_DATASETNAME = "vimmrc"
+
+_DESCRIPTION = """
+ViMMRC, a challenging machine comprehension corpus with multiple-choice questions,
+intended for research on the machine comprehension of Vietnamese text. This corpus
+includes 2,783 multiple-choice questions and answers based on a set of 417 Vietnamese
+texts used for teaching reading comprehension for 1st to 5th graders.
+"""
+
+_HOMEPAGE = "https://sites.google.com/uit.edu.vn/kietnv/datasets#h.1qeaynfs79d1"
+
+_LANGUAGES = ["vie"]
+
+_LICENSE = Licenses.UNKNOWN.value
```

Comment:
since the `HOMEPAGE` mentions this **"The corpus is freely available at our website for research purposes."**, can we append it to the `_LICENSE` value?

Reply:
and since I don't think the boundary is clear either, I'm leaning towards keeping it as `UNKNOWN` license and just append the text that I mentioned rather than altering it to `OTHER` license